### DISCUSSION TITLE
782

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#953] [Comments]
# 782

최근에 많은 분들을 뵙고 이야기 나누다가 빠르게 승진 중이신 분의 커리어 고민에 대해 이야기할 기회가 있었다.굉장히 큰 회사에서 나이나 연차에 비해 빠르게 승진과 리더십 기회를 얻은 분이셨다.최상위 리더의 평가만 좋은게 아니라, 같이 일하시는 주변 분들의 평가도 대단히 좋은 분이라서 "어떻게 하면 그 위치의 역할을 잘 할 수 있을까" 에 대해 고민 하시는 줄 알았지만, 그게 아니였다.오히려 너무 빠른 승진과 기회로 인해 비슷한 연차, 비슷한 나이대의 동료들과 거리가 멀어지고 있는 것에 대한 고민이였다.권한 혹은 리더십이 높아질수록 점점 조직의 사정을 더 알게 되기도 하고, 개인의 성과 보다는 팀의 성과, 팀원들의 성과, 조직에 더 도움이 되는 결정 등에 대해 고민하다보니 매일 하는 업무, 고민, 생각 등..

https://jojoldu.tistory.com/782

**Comment by devholic22** [comment]
향로님 오늘도 좋은 글 감사합니다!

**Reply by jojoldu** [reply]
잘 봐주셔서 감사합니다 ㅎㅎ

**Comment by kingbbode** [comment]
감사합니다 :)

**Reply by jojoldu** [reply]
ㅋㅋ 저도 감사합니다 뽀대님! 응원합니다 :)

**Comment by LenKIM** [comment]
좋은 글 감사합니다~

**Comment by Hoo93** [comment]
요즘 같은 시대에 20~30대 들에거 정말 필요한 마인드가 아닐까 싶네요.
많은 위로가 되는 글이었습니다.
좋은 글 감사합니다 !

**Comment by hojinDev** [comment]
좋은글 감사합니다.
때를 잘만나 밥 벌어먹고 살수있어 항상 감사함을 느낍니다.
